trojsten/submit/templates/trojsten/submit/protocol.html

```diff
@@ -5,45 +5,50 @@
-{% if compileLogPresent %}
-    <h4>Výstup z kompilátora</h4>
-    <pre>{{ compileLog }}</pre>
-{% endif %}
-{% if have_tests %}
-    <table class="table table-hover">
-        <thead>
-            <tr>
-                <th>Vstup</th>
-                <th>Výsledok</th>
-                <th>Čas</th>
-            </tr>
-        </thead>
-        <tbody>
-            {% for test in tests %}
-                <tr class="{%if test.result == "OK" %}success{% elif test.result == "IGN" %}warning{% else %}danger{% endif %}">
-                    <td>{{ test.name }}
-                        {% if test.showDetails %}
-                            <a data-toggle="collapse" href="#collapse{{ forloop.counter0 }}">Porovnanie výstupov</a>
+    {% if compileLogPresent %}
+        <h4>Výstup z kompilátora</h4>
+        <pre>{{ compileLog }}</pre>
+    {% endif %}
+    {% if have_tests %}
+        <div class="protocol-sets">
+        {% for test_set in test_sets.values %}
+            <div class="protocol-set">
+                <div class="protocol-box">
+                    <span class="protocol-set-name">
+                        {% if test_set.name == '00' %}
+                            sample
+                        {% else %}
+                            {{ test_set.name }}
                         {% endif %}
-                    </td>
-                    <td><span data-toggle="tooltip" data-placement="auto" title="{{ test.result }}">{{ submit_verbose_response|lookup:test.result }}</span></td>
-                    <td>{{ test.time }} ms</td>
-                </tr>
+                    </span>
+                    <span class="protocol-set-result">{{ test_set.ok }}/{{ test_set.tests|length }}</span>
+                </div>
 
-                    {% if test.showDetails %}
-                <tr>
-                <td colspan="3"  style="padding:0">
-                    <div class="collapse" id="collapse{{ forloop.counter0 }}">
-                        <pre><samp>{{ test.details }}</samp></pre>
-                    </div>
-                </td>
-                </tr>
-                   {% endif %}
-            {% endfor %}
-        </tbody>
-    </table>
-{% endif %}
+                {% for test in test_set.tests %}
+                <div class="protocol-box with-tooltip {% if test.showDetails %}protocol-has-details{% endif %} protocol-{%if test.result == "OK" %}ok{% elif test.result == "IGN" %}ign{% elif test.result == "WA" %}wa{% else %}tle{% endif %}"
+                    title="<b>{{ test.name }}</b><br>{{ submit_verbose_response|lookup:test.result }}{% if test.showDetails %}<br><small>Porovnanie výstupov pod tabuľkou</small>{% endif %}"
+                    data-input="{{ test.name }}">
+                    <span class="protocol-result">{{ test.result }}</span>
+                    <span class="protocol-time">{{ test.time }} ms</span>
+                </div>
+                {% endfor %}
+            </div>
+        {% endfor %}
+        </div>
```

Comment:
Toto sa stale scrolluje kazdy riadok zvlast. Pride mi prirodzenejsie mat to ako tabulku pomocou tagov `<th>` pre summary a `<td>` pre ostatne boxy.

Reply:
Vieš mi povedať, na čom si to testoval? v FF 68 sa to scrolluje naraz

Reply:
Chrome.

Reply:
No, neviem...

Desktop (Linux, responsive mode v devtools):
Chromium 76 - works
Firefox 68 - works

Andorid:
Chrome 78 - works
Firefox 68.2 - works

Reply:
Uz mi to funguje. Mozno som nieco zabudol pullnut, sorry.